components/structures/TopNav/topNav.module.css

```diff
@@ -19,6 +19,7 @@
   display: flex;
   align-items: center;
   width: 100%;
+  background-color: var(--ye-color-white);
```

Comment:
`.root` class already sets the background-color property, any case for setting it again for `.container`?